src/doc/using_visit/Preferences/Rendering_Options_Window.rst

```diff
@@ -160,6 +160,55 @@ To change the scalable rendering mode, click on either the **Auto**,
 **Always** or **Never** radio boxes under the **Use scalable rendering**
 label.
 
+ANARI Rendering
+~~~~~~~~~~~~~~~
+
+.. _Preferences-AnariRenderingOptions:
+
+.. figure:: images/AnariRenderingOptions.png
+
+    ANARI rendering options
+
+If VisIt_ is built with `ANARI <https://www.khronos.org/api/index_2017/anari/>`_ support by using the ``--anari`` option
```

Comment:
Please modify so that there is only one sentence per line, regardless of length.

https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/ContributingToDocumentation/index.html#about-line-length

Reply:
File updated to be one sentence per line